src/doc/unstable-book/src/compiler-flags/coverage-options.md

```diff
@@ -5,4 +5,7 @@ This option controls details of the coverage instrumentation performed by
 
 Multiple options can be passed, separated by commas. Valid options are:
 
-- `no-branch`, `branch` or `mcdc`: `branch` enables branch coverage instrumentation and `mcdc` further enables modified condition/decision coverage instrumentation. `no-branch` disables branch coverage instrumentation as well as mcdc instrumentation, which is same as do not pass `branch` or `mcdc`.
+- `block` (default): Disables branch coverage and MC/DC coverage.
```

Comment:
I would suggest to explain what we indeed enabled by using this flag, rather than what we don't enable.

Maybe something like this (I am not sure "statement blocks coverage" is the right definition though).
```suggestion
- `block` (default): Enables simple statement blocks coverage instrumentation.
```